src/python/bcc/libbcc.py

```diff
@@ -147,6 +147,10 @@
 lib.bpf_attach_perf_event.argtype = [ct.c_int, ct.c_uint, ct.c_uint, ct.c_ulonglong, ct.c_ulonglong,
         ct.c_int, ct.c_int, ct.c_int]
 
+from .perf import Perf
```

Comment:
could you move this import to the top of the file?

Reply:
@davemarchevsky Hi Dave,

I had tried putting it to top of file ( right after `import of ctypes` ) , but there is import dependency on `lib` with that change.

```>>> import bcc
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/site-packages/bcc/__init__.py", line 26, in <module>
    from .libbcc import lib, bcc_symbol, bcc_symbol_option, bcc_stacktrace_build_id, _SYM_CB_TYPE
  File "/usr/lib/python3.6/site-packages/bcc/libbcc.py", line 16, in <module>
    from .perf import Perf
  File "/usr/lib/python3.6/site-packages/bcc/perf.py", line 17, in <module>
    from .utils import get_online_cpus
  File "/usr/lib/python3.6/site-packages/bcc/utils.py", line 20, in <module>
    from .libbcc import lib
ImportError: cannot import name 'lib'
```

I can put it at top but after we define `lib`.

```diff --git a/src/python/bcc/libbcc.py b/src/python/bcc/libbcc.py
index 959296e3..ea19406b 100644
--- a/src/python/bcc/libbcc.py
+++ b/src/python/bcc/libbcc.py
@@ -16,6 +16,7 @@ import ctypes as ct
 
 lib = ct.CDLL("libbcc.so.0", use_errno=True)
 
+from .perf import Perf
 # keep in sync with bcc_common.h
 lib.bpf_module_create_b.restype = ct.c_void_p
 lib.bpf_module_create_b.argtypes = [ct.c_char_p, ct.c_char_p, ct.c_uint,
@@ -147,6 +148,9 @@ lib.bpf_attach_perf_event.restype = ct.c_int
 lib.bpf_attach_perf_event.argtype = [ct.c_int, ct.c_uint, ct.c_uint, ct.c_ulonglong, ct.c_ulonglong,
         ct.c_int, ct.c_int, ct.c_int]
 
+lib.bpf_attach_perf_event_raw.restype = ct.c_int
+lib.bpf_attach_perf_event_raw.argtype = [Perf.perf_event_attr(), ct.c_uint, ct.c_uint, ct.c_uint, ct.c_uint]
+
 lib.bpf_close_perf_event_fd.restype = ct.c_int
 lib.bpf_close_perf_event_fd.argtype = [ct.c_int]
```

Does that look fine ?

Thanks
Athira

Reply:
Sure, will move this import to begining of file after `lib` line